Celeste.Mod.mm/Patches/LevelLoader.cs

```diff
@@ -37,7 +38,7 @@ public patch_LevelLoader(Session session, Vector2? startPosition = default)
         public extern void orig_ctor(Session session, Vector2? startPosition = default);
         [MonoModConstructor]
         public void ctor(Session session, Vector2? startPosition = default) {
-            Logger.Log(LogLevel.Info, "LevelLoader", $"Loading map {session?.Area.GetSID() ?? "NULL"}");
+            Logger.Log(LogLevel.Info, "LevelLoader", $"Loading {session?.Area.GetSID() ?? "NULL"}");
```

Comment:
Not sure about all the "map" removals in general, but this one in particular I'm not a fan of. I think making it clear that the thing being loaded is a map is helpful, especially since to someone who doesn't know what the LevelLoader does, but knows that Level is the internal name for rooms, might think associate this with rooms.

Reply:
Original load logs looked like this:
```
[Everest] [Info] [LevelLoader] Loading level flowerhouse/QM2/5_SawrfingCastle
(just crashes on error lmao)
(no log on loadlevel success)
[Everest] [Warn] [LoadLevel] Failed loading level 12 (SID: flowerhouse/QM2/5_SawrfingCastle)
```

Proposed logs look like this:
```
[Everest] [Info] [LevelLoader] Loading flowerhouse/QM2/5_SawrfingCastle
[Everest] [Warn] [LevelLoader] Failed loading flowerhouse/QM2/5_SawrfingCastle
[Everest] [Verbose] [LoadLevel] Loading room 00 of flowerhouse/QM2/5_SawrfingCastle
[Everest] [Warn] [LoadLevel] Failed loading room 00 of flowerhouse/QM2/5_SawrfingCastle
```

The "map" addition was from my prior postcard PR to try and differentiate things since the original logs were so ambiguous. With the changes though, I think that room vs. no room makes a good enough distinction. The postcard warning has always used just the SID as well.